scripts/vscripts/ZombieReborn.lua

```diff
@@ -73,6 +75,12 @@ function OnPlayerDeath(event)
     if hAttacker:GetTeam() == CS_TEAM_T and hVictim:GetTeam() == CS_TEAM_CT then
         DoEntFireByInstanceHandle(hVictim, "runscriptcode", "Infect(nil, thisEntity, true)", 0, nil, nil)
     end
+
+    -- Infect Humans that died after first infection has started
+    if ZR_ROUND_STARTED and ZR_ZOMBIE_SPAWNED and hVictim:GetTeam() == CS_TEAM_CT then
+        --print("infecting suicide")
+        Infect(nil, hVictim, bSpawnType)
```

Comment:
Wouldn't this cause duplicate killfeed?

Reply:
It does, but SetTeam also duplicates the killfeed.

Reply:
Not if you setteam while they are actually dead. Try delaying it by one tick using EntFire.

Reply:
Fixed